src/software/pipeline/main_panoramaInit.cpp

```diff
@@ -229,7 +231,7 @@ class CircleDetector {
       int max_x = -1;
 
       /*Lookup possible radius*/
-      int start = std::max(0, int(level_radius) - uncertainty);
+      int start = std::max(min_radius, int(level_radius) - uncertainty);
```

Comment:
Could we also add a way to not be bigger than the diagonal?

Reply:
and a min/max also for the optical center like in the center within 20% of the image.

Reply:
The diagonal limit is implicit thanks to polar image construction